index.html

```diff
@@ -5156,17 +5156,38 @@ <h3>Overview</h3>
         and generating a JSON-LD document in expanded form for all
         <a>RDF literals</a>, <a>IRIs</a>
         and <a>blank node identifiers</a>.
+
         If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set to <code>true</code>,
         <a>RDF literals</a> with a
         <a>datatype IRI</a>
-        that equals <code>xsd:integer</code> or <code>xsd:double</code> are converted
-        to a <a>JSON numbers</a> and <a>RDF literals</a>
+        that equals
```

Comment:
This list markup inside a sentence style is a bit difficult to read.  I'd suggest rewording the changes and using a similar style to what is elsewhere in the spec.